wsagent/che-core-api-project/src/main/java/org/eclipse/che/api/vfs/search/impl/LuceneSearcher.java

```diff
@@ -242,15 +242,15 @@ private Query createLuceneQuery(QueryExpression query) throws ServerException {
         final String path = query.getPath();
         final String text = query.getText();
         if (path != null) {
-            luceneQuery.add(new PrefixQuery(new Term("path", path)), BooleanClause.Occur.MUST);
+            luceneQuery.add(new PrefixQuery(new Term("path", path)), MUST);
```

Comment:
I think we should keep BooleanClause.Occur.MUST way. It has much more descriptive.